Source/DataSources/PolylineGeometryUpdater.js

```diff
@@ -538,7 +538,7 @@ import Property from './Property.js';
      * Creates the dynamic updater to be used when GeometryUpdater#isDynamic is true.
      *
      * @param {PrimitiveCollection} primitives The primitive collection to use.
-     * @param {PrimitiveCollection|OrderedGroundPrimitiveCollection} groundPrimitives The primitive collection to use for ordered ground primitives.
+     * @param {PrimitiveCollection} groundPrimitives The primitive collection to use for ordered ground primitives.
```

Comment:
Since these docs are both the HTML users browse and our own code doc...is this now confusing for others on the team to read this and not know the other type it can take? 

I feel like the right way to fix this is for classes to not expose any public methods that take in any private properties. If a class like this needs it, then the whole class probably shouldn't be public?

Reply:
So I'm not sure removing this `OrderedGroundPrimitiveCollection` for the sake of fixing the broken 404 in the generated docs is a good idea.